src/components/PostList/PostList.jsx

```diff
@@ -1,3 +1,12 @@
-export const PostList = () => (
-  <>Put the list here</>
+import { PostInfo } from '../PostInfo';
+
+export const PostList = ({ posts }) => (
+  <ul className="PostList">
+    {posts && posts.map(post => (
```

Comment:
```suggestion
    {posts.map(post => (
```
we don't need this check cause posts are always an array